### DISCUSSION TITLE
Getting “command ‘markdown.showPreviewToSide’ not found” error on iPad

### DISCUSSION THREAD

**Discussion by tim-rohrer** [coder/code-server#3671] [Q&A] [answered]
I’m running `code-server` in a Docker container. I’m getting the subject error when I connect from iPad (it worked once, but then wouldn’t work any more) and try to do a markdown preview. I’ve also tried using the preview extension with the same result.

I’ve updated the server to the latest version, but no luck. 

![IMG_0139](https://user-images.githubusercontent.com/7756865/123341609-7d641680-d50b-11eb-885d-cdcf5806841c.jpeg)

Is anyone getting a preview to work? What are some things I can try next?

Thanks.

**Comment by jsjoeio** [comment, marked as answer]
Interesting 🤔 I'm seeing a similar error on my iPad.

https://user-images.githubusercontent.com/3806031/123674972-125e5c80-d7f7-11eb-9a91-d2f70d230f2a.MP4

![IMG_0260](https://user-images.githubusercontent.com/3806031/123674991-18543d80-d7f7-11eb-8a39-af766c018125.PNG)

I'm not sure why this is happening. I'm going to convert this to an issue for tracking

**Reply by jsjoeio** [reply]
https://github.com/cdr/code-server/issues/3686

**Comment by rumunius** [comment]
I also have this problem on code-server.

**Reply by jsjoeio** [reply]
Subscribe to this issue for updates: https://github.com/cdr/code-server/issues/3686
